tools/crates-validator/src/gen_profiles/mod.rs

```diff
@@ -0,0 +1,450 @@
+// Copyright 2023 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO(Cardosaum): Check how to export only the functions that are needed
```

Comment:
Add backlog issue to check all exported data structures and functions, limiting to only those strictly needed to be pub (idea)

Reply:
Will be addressed in #947